config.md

```diff
@@ -231,4 +231,17 @@ If a hook returns a non-zero exit code, then an error is logged and the remainin
 `args` and `env` are optional.
 The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://golang.org/pkg/os/exec/#Cmd).
 
+## Annotations
+
+Annotations are optional arbitrary non-identifying metadata that can be attached to containers.
```

Comment:
“arbitrary” and ”non-identifying” seem to conflict.  I'd expect we want to drop “non-identifying” and just missed it in the reroll that dropped labels.